cpp/cmake/Dependencies.cmake

```diff
@@ -38,8 +38,8 @@ else(DEFINED ENV{RAFT_PATH})
   set(RAFT_DIR ${CMAKE_CURRENT_BINARY_DIR}/raft CACHE STRING "Path to RAFT repo")
 
   ExternalProject_Add(raft
-    GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           c5030a0f9cc7f95ce4208febc89b77ccf251b60e
+    GIT_REPOSITORY    https://github.com/cjnolet/raft.git
+    GIT_TAG           fea-020-hdbscan
```

Comment:
This now needs to be updated in `cpp/cmake/thirdparty/get_raft.cmake` as opposed to here when merging branch-21.06 into the PR